### DISCUSSION TITLE
How to format text in the Album Description?

### DISCUSSION THREAD

**Discussion by hhanzo1** [LycheeOrg/Lychee#2510] [Q&A] [answered]
I've started to add info to my albums in the Description section but cannot format new lines.

It appears markdown does work but not new lines.  Everything appears on one line unless I put 2 new lines (but then there is an extra line when the album page is displayed).

What's the secret?

**Comment by ildyria** [comment, marked as answer]
It does follow Markdown. :)
```
text<space><space>  
test
```
Will be displayed on two lines.
https://www.markdownguide.org/basic-syntax/#line-break-best-practices

**Reply by hhanzo1** [reply]
Awesome. In my markdown editor, I just hit enter and it's a new line.

Thank you for the speedy response!
